docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -144,12 +319,36 @@ State and Transition Data Tables
 Model Assumptions and Limitations
 ---------------------------------
 
+.. todo:
+	
+	note neonatal copied prevalence and pregnancy adjustment here
+
 Validation Criteria
 -------------------
 
 References
 ----------
 
-Kassebaum NJ, GBD 2013 Anemia Collaborators. The Global Burden of Anemia. 
-Hematol Oncol Clin North Am. 2016 Apr;30(2):247-308. doi: 
-10.1016/j.hoc.2015.11.002
+.. [Kassebaum-et-al-2016]
+
+	View `Kassebaum et al. 2016` 
```

Comment:
This is slightly confusing (also, missing underscore) to have a separate line for the link.  I only did that for the appendices because they are distinct from the actual citation - i.e. we are mostly interested in the appendices, not the entire paper.

Instead, I would probably just make the entire paper title into a link, in the citation itself.